controllers/errors.controller.js

```diff
@@ -7,5 +7,7 @@ exports.handleCustomErrors = (err, req, res, next) => {
 exports.handlePsqlErrors = (err, req, res, next) => {
   if (err.code === "22P02") {
     res.status(400).send({ msg: "Bad Request" });
+  } else if (err.code === "23502") {
+    res.status(400).send({ msg: err.msg || "Bad Request" });
```

Comment:
You'll find that the error will only have a `msg` property if it's one of your custom Promise rejections. I'd have thought you'll be able to hardcode "Bad Request" here as the above error handler will have already handled this 👍